build.gradle

```diff
@@ -715,11 +714,7 @@ project('shared:controller-api') {
         compile project(':common')
         compile group: 'org.apache.commons', name: 'commons-lang3', version: commonsLang3Version
         compile group: 'commons-io', name: 'commons-io', version: commonsioVersion
-        compile "io.grpc:grpc-netty:" + grpcVersion
```

Comment:
Here and above, I see the dependency being removed but nothing is added. Where is this shaded dep coming from?

Reply:
It is coming from the protobuf.gradle . This will be applied fromthe controller-api project.
Let me know if there is a better way.